src/workflow/confirm.h

```diff
@@ -27,6 +28,11 @@
  * @return true if the user accepted, false if the user rejected.
  */
 bool workflow_confirm(const char* title, const char* body, bool longtouch, bool accept_only);
+enum ready workflow_confirm_async(
```

Comment:
please add a docstring, it's not obvious how to use it.

Reply:
ack